deployments/helm/proxy-nsmgr/templates/proxy-nsmgr.tpl

```diff
@@ -13,24 +13,38 @@ spec:
       labels:
         app: proxy-nsmgr-daemonset
     spec:
+      serviceAccount: nsmgr-acc
       containers:
         - name: proxy-nsmd
           image: {{ .Values.registry }}/{{ .Values.org }}/proxy-nsmd:{{ .Values.tag }}
           imagePullPolicy: {{ .Values.pullPolicy }}
           ports:
             - containerPort: 5006
               hostPort: 5006
+          env:
+            - name: INSECURE
+{{- if .Values.insecure }}
+              value: "true"
+{{- else }}
+              value: "false"
+{{- end }}
         - name: proxy-nsmd-k8s
           image: {{ .Values.registry }}/{{ .Values.org }}/proxy-nsmd-k8s:{{ .Values.tag }}
           imagePullPolicy: {{ .Values.pullPolicy }}
           ports:
-            - containerPort: 80
-              hostPort: 5005
+            - containerPort: 5005
+              hostPort: 80
```

Comment:
taking hostPort 80 seems crazy to me.

Reply:
Hello @tiswanso 
80 port was chosen in Interdomain Spec #714 and you can always change it with environment variable `PROXY_NSMD_K8S_REMOTE_PORT`
Also it very helpful for us because 80 port is opened by default in all clouds

Reply:
I realize this is to get the tests to pass and the rationale is to try and implement the tests' interdomain deployment so that proxy-nsmgr's NSR is at the domain's port 80.  But in the tests the domain is the public IP of the node with proxy-nsmgr and not really a service exposed in a typical way via DNS and a loadbalancer which is more the spirit of what's in the Interdomain spec.  

My point is that, it's a bad idea to assume NSM owns the nodes by grabbing well known ports __on the node__.  By taking a node's port 80, the installation is very likely to conflict with something in a real k8s deployment.  In pub clouds port 80 opened externally because it is typically mapped to an API gateway of some sort (e.g. nginx/k8s-ingress/istio-gateway).  

So, IMO, this looks really hacky/crazy to any operator taking a look at NSM helm charts.  There's no real reason to expose the service like this other than what you're doing to make it work in CI tests.